Nikita_Korzhavin/js9countDown.js

```diff
@@ -0,0 +1,14 @@
+function countDown(num) {
+  for (var i = num; i >= 0; i--) {
+    function time(i) {
+      setTimeout(function() {
+        console.log(i);
+      }, 1000 * (num - i));
+    }
+    var binded = time.bind(null, i);
+    binded(i);
```

Comment:
You could write it like this `time.bind(null, i)()` or even better `time.bind(null)(i)` which is called `currying`. 
Although I don't insist on changing it but strongly recommend you to read more about it since it is a part of functional approach and is widely used by JS programmers nowadays

Reply:
fixed